backend/app/api/endpoints/adapter/subscription_follows.py

```
@@ -76,6 +82,26 @@ def unfollow_subscription(
     )
 
 
+@router.patch("/{subscription_id}/follow/notification", status_code=status.HTTP_200_OK)
+def update_follow_notification(
+    subscription_id: int,
+    request: UpdateFollowNotificationRequest,
+    db: Session = Depends(get_db),
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)